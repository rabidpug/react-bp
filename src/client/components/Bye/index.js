import React from 'react';
import styles from './styles.scss';

const Bye = () => ( <div className={ styles.bye }>
  {'Bye'}
</div> );

export default Bye;
