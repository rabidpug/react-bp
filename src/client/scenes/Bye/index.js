import React from 'react';
import logo from '../../assets/logo.svg';
import styles from './styles.scss';

const Bye = () => ( <div className={ styles.bye }>
  <img src={ logo } />
  {'Bye'}
</div> );

export default Bye;
