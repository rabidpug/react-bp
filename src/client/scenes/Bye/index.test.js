import Bye from '.';
import React from 'react';
import { shallow, } from 'enzyme';

describe(
  'TopBar', () => {
    it(
      'Should render the TopBar', () => {
        const wrapper = shallow( <Bye /> );

        expect( wrapper ).toMatchSnapshot();
      }
    );
  }
);
