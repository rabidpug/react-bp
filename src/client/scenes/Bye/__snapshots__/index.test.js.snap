// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopBar Should render the TopBar 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Bye />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <img
          src="test-file-stub"
        />,
        "Bye",
      ],
      "className": "bye",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "src": "test-file-stub",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      "Bye",
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <img
            src="test-file-stub"
          />,
          "Bye",
        ],
        "className": "bye",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "src": "test-file-stub",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        "Bye",
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
